fighters/dedede/src/opff.rs

```diff
@@ -119,6 +123,9 @@ unsafe fn gordo_recatch(boma: &mut BattleObjectModuleAccessor, frame: f32, fight
             if fighter.status_frame() > 1 && fighter.status_frame() < 4{
                 ControlModule::reset_main_stick(boma);
             }
+            if StatusModule::is_situation_changed(fighter.module_accessor){
+                MotionModule::set_rate(fighter.module_accessor, 1.4);
+            }
```

Comment:
what are we doing here..? this will presumably just be overwritten by any motion rating done in the ACMD. maybe you are trying to solve a problem that is caused by placing motion rating calls inside an `if is_excute(agent)` conditional? Whatever it is, you probably do not need or want to set motion rate to 1.4 inside of status on `StatusModule::is_situation_changed`.